listenbrainz_spark/stats/__init__.py

```diff
@@ -145,6 +165,9 @@ def get_dates_for_stats_range(stats_range: str) -> Tuple[datetime, datetime]:
     elif stats_range == "month":
         from_offset = relativedelta(months=-1, day=1)  # first day of previous month
         to_offset = relativedelta(months=+1)
+    elif stats_range == "quarter":
+        from_offset = get_last_quarter_offset(latest_listen_date)
+        to_offset = relativedelta(months=+3)
```

Comment:
We had a 1 line discussion in IRC suggesting that we could also have the current quarter interpreted as "up-to-today". Do we also want this?

https://chatlogs.metabrainz.org/libera/metabrainz/2021-09-28/?msg=4865940&page=1

Reply:
That should be a different category of stats if we decide to add it. For now, its last calendar period for all time ranges.